### DISCUSSION TITLE
Are you opposed to Windows specific dependencies?

### DISCUSSION THREAD

**Discussion by jeanga** [starship/starship#1127] [General]
Hi,
I am looking into usernames.rs (because it is broken for Windows users).

There is more than one way to fix it but the best approach (IMHO) would be to introduce a dependency to the winapi crate on Windows.
Before I embark into this path, I'd like to ask if you would be uncomfortable with adding this dependency in Cargo.toml:
```
[target.'cfg(windows)'.dependencies]
winapi = "0.3.8"
```

Thank you for your feedback!
Jean

**Comment by matchai** [comment]
As long as it is only added to the Windows build, I see no issue with adding such a dependency. 😊

**Comment by Francinededans** [comment]
Bonsoir comment vous allez ??
